### PR TITLE
fix: accept both index and manifest digests when creating pyxis records

### PR DESCRIPTION
**Pull request by ralphbean**
We need both when creating a multi-arch image. Both manifest_schema2_digest and manifest_list_digest should be supplied.

### CODE REVIEW

**Comment by ralphbean** [comment]
When merged, this change needs to be promoted alongside https://github.com/redhat-appstudio/release-service-catalog/pull/388

File: utils/get-image-architectures [hunk 1]
```diff
@@ -2,8 +2,8 @@
 #
 # script:      get-image-architectures
 # 
-# description: This script fetches all architectures for a passed image.
-#              It will output each arch on its own line.
+# description: This script fetches all architectures and their digests for a passed image.
+#              It will output each arch on its own line, base64 encoded.
```
Comment (johnbieren) [inline utils/get-image-architectures:6]: Why are we base64 encoding digests?

Comment (davidmogar) [reply utils/get-image-architectures:6]: Not sure that I understand correctly, but you can for sure iterate over lines with bash. You could do something like the following:
```
~ % cat lines 
{"line": 1, "foo": "bar", "baz": "qux"}
{"line": 2, "foo": "bar", "baz": "qux"}
{"line": 3, "foo": "bar", "baz": "qux"}
{"line": 4, "foo": "bar", "baz": "qux"}
{"line": 5, "foo": "bar", "baz": "qux"}
~ % cat script.sh 
#!/bin/bash

cat lines | while IFS= read -r line; do
    echo "Processing line $(echo $line | jq '.line')"
done
~ % ./script.sh 
Processing line 1
Processing line 2
Processing line 3
Processing line 4
Processing line 5
```

Comment (ralphbean) [reply utils/get-image-architectures:6]: Ack, will refactor to use that model.

Comment (ralphbean) [reply utils/get-image-architectures]: Done in [09cfc9c](https://github.com/redhat-appstudio/release-service-utils/pull/134/commits/09cfc9cdd0d27422ec78d0fb7d70bca7c2cb078e) and https://github.com/redhat-appstudio/release-service-catalog/pull/388

**Review by johnbieren — APPROVED** [review]
needs rebase but lgtm

**Comment by ralphbean** [comment]
@johnbieren rebased on main.

**Review by johnbieren — APPROVED** [review]

**Pull request merged**